src/core/output.ts

```diff
@@ -14,7 +14,7 @@ type StepResultsHelper<InterestedIn extends StepName> = {
 	'parse':       Out<'parse'>
 	'normalize':   StepResultsHelper<'parse'> & Out<'normalize'>
 	'dataflow':    StepResultsHelper<'normalize'> & Out<'dataflow'>
-	'slice':       StepResultsHelper<'dataflow'> & Out<'slice'>
+	'ai':          StepResultsHelper<'normalize'> & StepResultsHelper<'dataflow'> & Out<'ai'>
```

Comment:
```suggestion
	'ai':          StepResultsHelper<'dataflow'> & Out<'ai'>
```
The `normalize` step is included recursively